templates/vue/src/components/Lightbox/media/VideoMedia/UrlVideoMedia.vue

```diff
@@ -3,6 +3,8 @@
     <video
       ref="video"
       controls
+      muted
```

Comment:
Just wondering why we want to mute all videos. Could this end up being confusing for a user?

Reply:
I'm not sure why the original author included this change either. I removed it and all tests are still passing. I think we should keep it removed.